### PR TITLE
You can now see the food in the microwave

### PR DESCRIPTION
**Pull request by Mothblocks**
https://user-images.githubusercontent.com/35135081/192945375-e45cb351-afa3-4085-abf9-b90ed5dee166.mp4

## Why It's Good For The Game

he forgot

## Changelog

:cl:
qol: You can now see food inside the microwave.
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
that. is huge.

**Comment by Time-Green** [comment]
Can you make them swivel around?

**Comment by ChristianDenDanske** [comment]
Finally now I can charge my PDA with greater detail.

**Comment by bruhman8ghughug** [comment]
WHAT WAS HE COOKING?

**Comment by Time-Green** [comment]
> > Can you make them swivel around?
> 
> I really wanted to but doing this with the way its set up would've been out of my field of expertise with BYOND rendering

Sad but yeah I understand

File: code/modules/food_and_drinks/machinery/microwave.dm [hunk 1]
```diff
@@ -31,11 +33,14 @@
 
 /obj/machinery/microwave/Initialize(mapload)
 	. = ..()
+
 	wires = new /datum/wires/microwave(src)
 	create_reagents(100)
 	soundloop = new(src, FALSE)
 	set_on_table()
 
+	update_icon_state()
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/machinery/microwave.dm:42]:
```suggestion
	update_appearance()
```

**Pull request merged**